### ISSUE TITLE
[BUG] wrong format for datetime-local component value

### ISSUE THREAD

**Issue by t0nka** [TiagoSilvaPereira/vemto-issues#97]
**Describe the bug**
the form-inputs.blade.php file of cruds contains wrong formated datetime fields

**To Reproduce**
1. create a model with datetime field
2. create a crud app for the model
3. create blade files
4. See error in the browser 

**Expected behavior**
as seen on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local the datetime-local input field needs a 2-digit-hour representation like 08:45 (H:i) but the generate code contains 8:45 aka G:i

**Example**

correct
```
<x-inputs.group class="col-sm-12">
    <x-inputs.datetime
        name="gueltig_bis"
        label="Gueltig Bis"
        value="{{ old('gueltig_bis', ($editing ? optional($nachricht->gueltig_bis)->format('Y-m-d\TH:i') : '')) }}"
    ></x-inputs.datetime>
</x-inputs.group>
```

wrong:
```
    <x-inputs.group class="col-sm-12">
        <x-inputs.datetime
            name="gueltig_von"
            label="Gueltig Von"
            value="{{ old('gueltig_von', ($editing ? optional($nachricht->gueltig_von)->format('Y-m-d\TG:i') : '')) }}"
        ></x-inputs.datetime>
```

**Comment by TiagoSilvaPereira** [comment]
Hi @t0nka thank you for reporting, I'll fix it for the next Vemto version

**Comment by TiagoSilvaPereira** [comment]
Fixed on version 0.8.1
